pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -109,6 +109,11 @@ xdsServer:
   dataplaneStatusFlushInterval: 10s # ENV: KUMA_XDS_SERVER_DATAPLANE_STATUS_FLUSH_INTERVAL
   # Backoff that is executed when Control Plane is sending the response that was previously rejected by Dataplane
   nackBackoff: 5s # ENV: KUMA_XDS_SERVER_NACK_BACKOFF
+  # A delay between proxy terminating a connection and the CP trying to deregister the proxy.
+  # It is used only in universal mode when you use direct lifecycle.
+  # Disabling this may cause race conditions that one instance of CP removes proxy object
+  # while proxy is connected to another instance of the CP.
+  dataplaneDeregistrationDelay: 10s # ENV: KUMA_XDS_DATAPLANE_DEREGISTRATION_DELAY
```

Comment:
Just one question, In the task it says:
> Delay unregistering DP to Insight flush interval * 2 and when unregistering, look into Insight and see if there is no new subscription.

and it looks like the flush interval is 10s, is it the correct default?

Reply:
Good question. I found a way to lower this latency by forcing sink flush on CP connect (first point from the PR), so this delay is no longer correlated with insight flush interval.
Technically for the simple reconnect probably something like ~2s delay would be enough, but I'd like to keep 10s and make it a configuration. If it's too low, you can always prolong it, or if you find this too long, you can shorten it.